.eslintrc.js

```diff
@@ -6,16 +6,24 @@
 'use strict'
 
 module.exports = {
-  extends: '@newrelic',
+  extends: ['@newrelic', 'plugin:jsdoc/recommended'],
+  plugins: ['jsdoc'],
   rules: {
-    'consistent-return': 'off'
+    'consistent-return': 'off',
+    'jsdoc/require-jsdoc': 'off'
   },
   overrides: [
     {
       files: ['newrelic.js'],
       rules: {
         'header/header': ['off']
       }
+    },
+    {
+      files: ['./lib/shim/*.js', 'lib/transaction/handle.js', 'api.js'],
+      rules: {
+        'jsdoc/require-jsdoc': 'warn'
```

Comment:
Wondering if this should be 'error'.  Easy enough to change later.

Reply:
Switching this to 'error' highlights 40 errors. Is that acceptable? Should I resolve them as part of this PR?

Reply:
Ah, given all the examples ended up with jsdoc I assumed this was covered. Lets land as warn then and then follow-up with switching to error and fixing things.

Reply:
Can do 👍